pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "alpineer"
-version = "0.1.4"
+version = "0.0.0"
```

Comment:
The versioning will be handled automatically? Or we want to reset it with this PR?

Reply:
It will be handled automatically. Internally it looks at git tags.